### PR TITLE
Add unplayed queue limit per-IP

### PR DESCRIPTION
**Pull request by Veltas**
- Adds unplayed queue limit per-IP.
- On refused uploads redirects to uploaderror.php, I have not written this page, you will do a better job keeping it consistent than I can (HTML is not my thing).
- Currently silently ignores extra queued YouTube vids when they're being added to the queue.
- I added code to delete files at the point when they're refused, so we don't leave them (otherwise they're only deleted after being played)
- I set a `final` variable for `max_buckets` in `ProcessQueue.java`. I went for 4, the magic number might be something else but I'd try 4 first.

I compiled and ran the code with these changes, so afaik nothing's broken.

### CODE REVIEW

**Comment by Veltas** [comment]
As clarification, `music-get.jar` is obsolete _in the repo_, not obsolete in general.

**Comment by mcnutty26** [comment]
Is there a reason not to check for rejection on submission of a youtube link? The only time it's OK to be silently ignoring submissions is when a YT download completes but the user has queued up more items in the mean time (I think)

**Comment by Veltas** [comment]
I agree, and thought about this later today while doing something else. Merge this PR and I'll make another to add that check if you want. It's an additional check, just for UX purposes, so I can do it separately when this is added.

**Comment by Veltas** [comment]
I was mainly concerned with making sure nobody could "break" the rule code at first.

**Comment by mcnutty26** [comment]
OK cool, LGTM.

File: src/main/java/mcnutty/music/get/ProcessQueue.java [hunk 1]
```diff
@@ -11,16 +11,29 @@
     public ArrayList<QueueItem> bucket_played;
     public ArrayList<QueueItem> bucket_youtube;
 
+    private static final int max_buckets = 4;
+
```
Comment (zed0) [inline src/main/java/mcnutty/music/get/ProcessQueue.java:15]: IMO this should be a command line/configuration file option if we can make it one.

Comment (Veltas) [inline src/main/java/mcnutty/music/get/ProcessQueue.java:15]: I agree, although I might not get to it

Comment (Veltas) [inline src/main/java/mcnutty/music/get/ProcessQueue.java:15]: Recommend submitting an issue requesting it

Comment (Veltas) [inline src/main/java/mcnutty/music/get/ProcessQueue.java:15]: NM, submitted a request for better config.ini: https://github.com/mcnutty26/music-get/issues/28

**Pull request merged**